config/default/config_ignore.settings.yml

```diff
@@ -2,10 +2,17 @@ ignored_config_entities:
   - 'core.entity_view_display.node.hs_*.default'
   - 'core.entity_form_display.node.hs_*'
   - 'menu_position.menu_position_rule.*'
-  - 'google_analytics.settings'
+  - google_analytics.settings
```

Comment:
missing quotes

Reply:
thats how it exported. i think it only needs the `'` if there is a `:` or `*`